src/cocaine-app/sched/recover_starter.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import time
+
+import jobs
+from mastermind_core.config import config
+import storage
+
+
+
```

Comment:
Fixed in https://github.com/yandex/mastermind/pull/433/commits/934c87a29040c96b1422713157e19689a8ce9fcd